<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MorseCode</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Text to Morse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Morse to Text</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the conversion</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="plainText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="morseCode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Morse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
